starknet_py/net/account/account.py

```diff
@@ -95,12 +81,14 @@ def __init__(
                 raise ValueError(
                     "Either a signer or a key_pair must be provided in Account constructor."
                 )
+            if chain is None:
+                raise ValueError("One of chain or signer must be provided.")
```

Comment:
Let's make sure that tests cover this properly after it was moved.

Reply:
Yup, they are
![image](https://user-images.githubusercontent.com/61014013/224635097-bda5be10-a40c-4873-8bc3-3aa22eda23c7.png)